content/_data/redirects.yaml

```diff
@@ -10,15 +39,89 @@
   to: /accoutrement/
 - from: /accoutrement-type/
   to: /accoutrement/
-- from: /community/
-  to: /birds/
-- from: /services/design-systems-training/
-  to: /workshops/resilient-systems/
-- from: /talks/resilient-systems/
-  to: /workshops/resilient-systems/
-- from: /talks/advanced-css-workshop/
-  to: /workshops/advanced-css/
-- from: /speaking/
-  to: /services/speaking/
-- from: /work/services/
-  to: /services/
+
+- from: /open-source/accoutrement/
+  to: /accoutrement/
+- from: /open-source/compass/
+  to: /compass/
+- from: /open-source/csswg/
+  to: /csswg/
+- from: /open-source/django/
+  to: /django/
+- from: /open-source/helium/
+  to: /helium/
+- from: /open-source/nunjucks/
+  to: /nunjucks/
+- from: /open-source/pip/
+  to: /pip/
+- from: /open-source/plone/
+  to: /plone/
+- from: /open-source/sass/
+  to: /sass/
+- from: /open-source/virtualenv/
+  to: /virtualenv/
+- from: /open-source/warehouse/
+  to: /warehouse/
+
+- from: /podcasts/
+  to: /tags/podcast/
+- from: /podcasts/talkscript/
+  to: /2018/09/06/talkscript/
+- from: /podcasts/thunder-nerds/
+  to: /2019/03/26/thunder-nerds/
+- from: /podcasts/versioning/
+  to: /2016/10/18/versioning/
+- from: /podcasts/vov-040/
+  to: /2018/12/04/vov-040/
+- from: /podcasts/vov-068/
+  to: /2019/07/02/vov-068/
+
+- from: /mozdev/
+  to: /tags/mozilla-developer-channel/
+- from: /mozdev/css-is-weird/
+  to: /2019/10/03/css-is-weird/
+- from: /mozdev/display-multi/
+  to: /2019/10/28/display-multi/
+- from: /mozdev/grids/
+  to: /2019/10/18/grids/
+- from: /mozdev/inactive-css/
+  to: /2019/10/21/inactive-css/
+- from: /mozdev/list-markers/
+  to: /2019/10/01/list-marker/
+- from: /mozdev/overflow-wrap/
+  to: /2019/11/06/overflow-wrap/
+- from: /mozdev/revert/
+  to: /2019/10/09/revert/
+- from: /mozdev/scroll-snap/
+  to: /2019/10/31/scroll-snap/
+- from: /mozdev/shape-editor/
+  to: /2019/11/25/shape-editor/
+- from: /mozdev/subgrid-cards/
+  to: /2019/10/15/subgrid-cards/
+- from: /mozdev/subgrid-forms/
+  to: /2019/10/16/subgrid-forms/
+- from: /mozdev/supports-selector/
+  to: /2019/11/20/supports-selector/
+
+- from: /blog-index/
+  to: /blog/
+- from: /2017/1/3/initial-letter/
+  to: /2017/01/03/initial-letter/
+- from: /2017/1/5/git-rebase/
+  to: /2017/01/05/git-rebase/
+- from: /2017/1/11/typography/
+  to: /2017/01/11/typography/
+- from: /2017/1/16/color/
+  to: /2017/01/16/color/
+- from: /2017/1/19/design-using-data/
+  to: /2017/01/19/design-using-data/
+- from: /2017/1/26/we-are-not-neutral/
+  to: /2017/01/26/we-are-not-neutral/
+- from: /2017/1/27/parallax/
+  to: /2017/01/27/parallax/
+- from: /2017/1/31/css-day/
+  to: /2017/01/31/css-day/
+
+# This was an already-broken link from the old OddSite...
+- from: /2015/08/14/coachhub-study/
+  to: /work/coachhub/
```

Comment:
This seems *super* edge-case (if someone somehow bookmarked a broken link -- which hasn't worked for years -- from a 2015 blog post?), but I guess it doesn't hurt.

Reply:
I was thinking if it were somehow indexed by google ... but yeah, not a huge deal.